apps/agentic-outer-dag/CLAUDE.md

```
@@ -0,0 +1,73 @@
+- running `resolve_pr_comments`,
+- posting Linear handoff comments.
+
+Safe stop-after values for Phase 1:
+- `dispatching_ticket_to_pr` — validate the existing-PR guard before any `linear_ticket_2_pr` dispatch.
+- `waiting_for_coderabbit` — stop before `dispatching_resolve_pr_comments`.
+
+Do not proceed past `dispatching_resolve_pr_comments` in Phase 1.
+
+Steps:
+1. Preview only:
+   ```bash
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)